meeting-notes/2019/Q2/2019-05-20--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,138 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 May 20, 2019
+
+- **Lead:** @jacobheun
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @jimpick
+  - @auhau
+  - @achingbrain
+  - @vmx
+  - @prabhakar-poudel
+- **Recording:** https://youtu.be/MXEx74Kyy60
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@jacobheun
+ - Done:
+   - Identify Push support in libp2p:
+     - Draft PR to Add identify push protocol in js-libp2p-identify [libp2p-identify#53](https://github.com/libp2p/js-libp2p-identify/pull/53)
+     - Integration of new identify API into Switch [libp2p-switch#338](https://github.com/libp2p/js-libp2p-switch/pull/338)
+       - Push support will be added later, once AutoNat is supported in js.
+   - Worked on the demo nodejs and browser apps for IPFS Camp Course
+ - Blocked:
+ - Next:
+   - IPFS Camp Course presentation and finish demo apps in js
+   - Finish Identify Push integration
+   - Start on AutoNat
+     - Will produce a Spec for it as a part of this effort.
+   - Go through infrastructure inventory of libp2p with Ops
+
+@vasco-santos (not attending)
+ - Done:
+   - Initial implementation of `js-gossipsub` PR got merged 🚀
+     - [ChainSafe/gossipsub-js#15](https://github.com/ChainSafe/gossipsub-js/pull/15)
+   - new implementation for the `interface-connection` proposal almost ready for review
+     - [libp2p/interface-connection#29#issuecomment-491262150](https://github.com/libp2p/interface-connection/pull/29#issuecomment-491262150)
+   - Data Terra Nemo
+ - Blocked: N/A
+ - Next:
+   - get `interface-connection` PR with the new proposal ready for review
+   - enable gossipsub interop tests and integration in `js-libp2p`
+   - more async iterators work
+
+@dirkmc
+ - Done:
+   - refs & refs-local endpoints released
+   - Garbage Collector locking first impl [js-ipfs#2022](https://github.com/ipfs/js-ipfs/pull/2022)
+ - Blocked:
+ - Next:
+   - Tests for GC locking
+
+@auhau (Adam)
+ - Done:
+   - PR for migration tool ready to review [AuHau/js-ipfs-repo-migrations#1](https://github.com/AuHau/js-ipfs-repo-migrations/pull/1)
+   - Initial integration to js-ipfs [js-ipfs#2044](https://github.com/ipfs/js-ipfs/pull/2044)
+ - Blocked:
+   - Guidance on how to continue with `js-ipfs-repo-migrations` (move under ipfs org?) 
```

Comment:
@auhau are you still looking for guidance on this?

Reply:
Well yes, but I assume that the guidance will hopefully come with the review of the PRs?